ballerina/rag-types.bal

```diff
@@ -78,7 +78,7 @@ public type MetadataFilters record {|
 # Supports precise search operations by combining vector similarity with metadata conditions.
 public type VectorStoreQuery record {|
     # The vector to use for similarity search
-    Embedding embedding;
+    Embedding embedding?;
```

Comment:
Shall we explain this nillable scenario in the APOI docs, otherwise users will get confuse

Reply:
Fixed in [754d891](https://github.com/ballerina-platform/module-ballerina-ai/pull/70/commits/754d891c64938302aa11361cd13554fc1aa711c8)